ui/components/app/edit-gas-popover/edit-gas-popover.component.js

```diff
@@ -41,7 +41,8 @@ export default function EditGasPopover({
   const dispatch = useDispatch();
   const showSidebar = useSelector((state) => state.appState.sidebar.isOpen);
 
-  const showEducationButton = mode === EDIT_GAS_MODES.MODIFY_IN_PLACE;
+  const showEducationButton =
+    mode === EDIT_GAS_MODES.MODIFY_IN_PLACE && process.env.SHOW_EIP_1559_UI;
```

Comment:
Since we have an issue open to audit `SHOW_EIP_1559_UI` usage, I used this flag which we'll need to update to `isEIP1559Transaction` in the future.